positron/modules/atom_browser_web_contents.js

```diff
@@ -33,6 +45,10 @@ let WebContents_prototype = {
     this._browserWindow._domWindow.location = url;
   },
 
+  getTitle: function() {
+    return "/* stub */";
```

Comment:
maybe just return the empty string or null, and console.warn("not implemented")?